R/Chrome.R

```diff
@@ -415,7 +415,7 @@ chr_launch <- function(
         bin,
         chrome_args,
         echo_cmd = TRUE,
-        supervise = TRUE
+        supervise = FALSE
```

Comment:
I don't understand why `supervise` needs to be set to `FALSE`

Reply:
A big mistake... 
This is a leftover from my test on previous issue. Sorry. 🤦‍♂ 
That is what reviews are for I guess.... 🤷‍♂

Reply:
Glad to help!